### PR TITLE
[8357] replace berlin polygon with germany and simplify condition for dasbhoard location menu item

### PR DESCRIPTION
**Pull request by goapunk**
**Tasks**
- [x] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: adhocracy-plus/templates/a4dashboard/base_dashboard_project.html [hunk 1]
```diff
@@ -42,28 +42,16 @@
                 </div>
                 <ul class="dashboard-nav__pages collapse{% if not project.id in closed_accordions.0 %} show{% endif %}" id="dashboardCollapseProj" aria-hidden="false">
                     {% for item in dashboard_menu.project %}
-			{% if item.label != "Location" %}
```
Comment (goapunk) [inline adhocracy-plus/templates/a4dashboard/base_dashboard_project.html:45]: simplified this a little to avoid the code duplication

**Review by m4ra — APPROVED** [review]
thanks!

**Comment by m4ra** [comment]
@goapunk any idea why the build check is on hold?

**Comment by goapunk** [comment]
@m4ra repushed, hopefully this fixes the ci, sometimes it seems to get stuck

**Pull request merged**